jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/PatternRule.java

```diff
@@ -59,7 +59,7 @@ public Processor matchAndApply(Processor input) throws IOException
      * @return the possibly wrapped {@code Request} and {@code Processor}
      * @throws IOException if applying the rule failed
      */
-    protected abstract Processor apply(Processor input) throws IOException;
```

Comment:
Change line 59 to:
`     * @return the possibly wrapped {@code Request} and {@code Handler}`

Or use `{@link ...}` instead of `{@code ...}`?